tools/cli/namespaceUtils.go

```diff
@@ -388,7 +388,7 @@ func initializeDynamicConfig(
 
 	// the done channel is used by dynamic config to stop refreshing
 	// and CLI does not need that, so just close the done channel
-	doneChan := make(chan struct{})
+	doneChan := make(chan interface{})
 	close(doneChan)
```

Comment:
the done channel only aims to be closed, so `chan struct{}` is better then `chan interface{}`

Reply:
It has data now. Interrupt signal will go there in case `temporal.InterruptCh is used`.